default/python/universe_generation/planets.py

```diff
@@ -1,9 +1,10 @@
-import sys
 import random
+import sys
+
 import freeorion as fo
-import util
-import universe_tables as tables
 
+import universe_tables as tables
+import util
 
```

Comment:
Same here...

Reply:
IMHO instead of pointing same issue multiple times, just leave one comment with `and in other places`.